vrtUtil/vrtUtil.js

```diff
@@ -0,0 +1,77 @@
+const path = require('path');
+const fs = require('fs');
+const ImageDiff = require('./ImageDiff');
+
+const VISUAL_REGRESSION_THRESHOLD = 0.01;
+
+const getAbsolutePathForScreenshot = (type, testFixture, testName) => (
+  path.join('screenshots', type, testFixture, `${testName}.png`).normalize()
+);
+
+const getDiffImageName = (imagePath) => {
+  return path.join(
+    path.dirname(imagePath),
+    `${path.basename(imagePath, path.extname(imagePath))}-diff.png`,
+  );
+};
+
+const doVisualRegression = async (testObject, name, updateScreenshot) => {
+  // set window size so screenshots are consistent
+  await testObject.resizeWindow(800, 800);
```

Comment:
We may need to consider putting the window back to the original size at the end of this method

Reply:
I think since the vrt is only used in the uiDemo spec now outside of the parity-v3 suite, this shouldn't be a problem